op-challenger/fault/agent.go

```diff
@@ -94,6 +94,17 @@ func (a *Agent) step(claim Claim, game Game) error {
 	if claim.Depth() != a.maxDepth {
 		return nil
 	}
+
+	if game.AgreeWithClaimLevel(claim) {
+		a.log.Warn("Agree with leaf claim, skipping step", "claim_depth", claim.Depth(), "maxDepth", a.maxDepth)
+		return nil
+	}
+
+	if claim.Countered {
+		a.log.Warn("Claim already stepped on", "claim_depth", claim.Depth(), "maxDepth", a.maxDepth)
```

Comment:
I don't think this are unusual enough to be a `Warn` - probably should just be `Info` level.